patterns/2-structured/base-documentation.md

```diff
@@ -137,10 +137,7 @@ started right away: [README-template.md](templates/README-template.md),
 ## Authors
 
 * Isabel Drost-Fromm
-
-## Acknowledgments
-
-* Katie Schueths - for adding the `COMMUNICATION.md` concept
+* Katie Schueths
```

Comment:
@kschueths very happy to list your here as author, rather than under Acknowledgements.

Would it be fine for you if we still add the below info next to your name, to highlight which part of the pattern you contributed? This might help readers in case they have questions and want to contact the respective authors.
> - for adding the `COMMUNICATION.md` concept

Reply:
Hi @kschueths . Thanks again for your changes here. 

Could you review the feedback above? Thanks.

Reply:
@spier. That sounds good.